dbsync.py

```diff
@@ -489,26 +490,33 @@ def dbsync_init(mc, from_gpkg=True):
     base_schema_exists = _check_schema_exists(conn, config.db_schema_base)
     modified_schema_exists = _check_schema_exists(conn, config.db_schema_modified)
 
-    try:
-        server_info = mc.project_info(config.mergin_project_name)
-    except ClientError as e:
-        raise DbSyncError("Mergin client error: " + str(e))
-
     gpkg_full_path = os.path.join(config.project_working_dir, config.mergin_sync_file)
-    if not os.path.exists(config.project_working_dir):
-        # download the Mergin project
-        if modified_schema_exists and base_schema_exists:
-            db_proj_info = _get_db_project_comment(conn, config.db_schema_base)
-            if not db_proj_info:
-                raise DbSyncError("Base schema exists but missing which project it belongs to")
-            print("Download version " + db_proj_info["version"] + " of Mergin project " + config.mergin_project_name +
-                  " to " + config.project_working_dir)
+    if modified_schema_exists and base_schema_exists:
+        # this is not a first run of db-sync init
+        db_proj_info = _get_db_project_comment(conn, config.db_schema_base)
+        if not db_proj_info:
+            raise DbSyncError("Base schema exists but missing which project it belongs to")
+
+        # make sure working directory contains the same version of project
+        if not os.path.exists(config.project_working_dir):
+            print(f"Downloading version {db_proj_info['version']} of Mergin project {config.mergin_project_name} "
+                  f"to {config.project_working_dir}")
             mc.download_project(config.mergin_project_name, config.project_working_dir, db_proj_info["version"])
         else:
-            print("Download latest Mergin project " + config.mergin_project_name + " to " + config.project_working_dir)
+            local_version = _get_project_version()
```

Comment:
Can you please add a print here that we already have a working directory with version XYZ?